src/types/Profiles.cpp

```diff
@@ -44,11 +52,13 @@ void Profiles::generateRandomPerfValues(unsigned long int seed, int lower_bound,
     throw std::invalid_argument(
         "Lower bound must be lower than the upper bound.");
   }
-  srand(seed);
+  std::mt19937 gen(seed);
+  std::uniform_real_distribution<> dis(lower_bound, upper_bound);
+
   for (int j = 0; j < pt_[0].size(); j++) {
     std::vector<float> r_vect;
     for (int i = 0; i < pt_.size(); i++) {
-      r_vect.push_back(getRandomUniformFloat(seed, lower_bound, upper_bound));
```

Comment:
same comment here as above the random changes should be made in utils

Reply:
done